hacking/aws_config/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains definitions to prepare an AWS account for running
```

Comment:
It does appear to be in docs now: docs/docsite/rst/dev_guide/testing_integration.rst